### PR TITLE
Added extra details about conversions.

### LINKED ISSUE #9766: double is missing.
There is not 'double' in the list. It is necessary to explain why. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 86308a57-1418-cf16-94f7-d840d56f6dda
* Version Independent ID: cdb096f4-7498-2db7-ad2d-cd7c794b0bee
* Content: [Implicit numeric conversions table - C# Reference](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/implicit-numeric-conversions-table)
* Content Source: [docs/csharp/language-reference/keywords/implicit-numeric-conversions-table.md](https://github.com/dotnet/docs/blob/master/docs/csharp/language-reference/keywords/implicit-numeric-conversions-table.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by ghost**
## Summary

Emphasized nonlisted datatypes at from and to columns.

Changed the order of char type to increase readability.

Fixes #9766

### CODE REVIEW

File: docs/csharp/language-reference/keywords/implicit-numeric-conversions-table.md [hunk 1]
```diff
@@ -18,24 +18,26 @@ The following table shows the predefined implicit conversions between .NET numer
...
 ## Remarks  
 
 - Any [integral type](integral-types-table.md) is implicitly convertible to any [floating-point type](floating-point-types-table.md).
 
 - Precision but not magnitude might be lost in the conversions from `int`, `uint`, `long`, or `ulong` to `float` and from `long` or `ulong` to `double`.  
   
-- There are no implicit conversions to the `char` type.  
+- There are no implicit conversions to the `bool`, `char`, `byte` and `sbyte` types.  
+
+- There are no implicit conversions from the `char`, `double` and `decimal` types.
```
Comment (pkulikov) [inline docs/csharp/language-reference/keywords/implicit-numeric-conversions-table.md]: If the bool type is mentioned in the previous point, it should appear here as well; to be consistent.

Comment (volkan-yazilim) [reply docs/csharp/language-reference/keywords/implicit-numeric-conversions-table.md]: 'bool' is removed.

File: docs/csharp/language-reference/keywords/implicit-numeric-conversions-table.md [hunk 2]
```diff
@@ -18,24 +18,26 @@ The following table shows the predefined implicit conversions between .NET numer
...
 
 - Precision but not magnitude might be lost in the conversions from `int`, `uint`, `long`, or `ulong` to `float` and from `long` or `ulong` to `double`.  
   
-- There are no implicit conversions to the `char` type.  
+- There are no implicit conversions to the `char`, `byte` and `sbyte` types.  
+
+- There are no implicit conversions from the `char`, `double` and `decimal` types.
   
-- There are no implicit conversions between the `float` and `double` types and the `decimal` type.  
+- There are no implicit conversions between `decimal` type and the `float` type or the `double` type.
```
Comment (rpetrusha) [inline docs/csharp/language-reference/keywords/implicit-numeric-conversions-table.md]:
```suggestion
 - There are no implicit conversions between the `decimal` type and the `float` or `double` types.  
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you, @volkan-yazilim, for reordering the numeric types nad for adding detail about the conversions. I've suggested a change that I think adds clarity to the sentence on decimal conversions.

File: docs/csharp/language-reference/keywords/implicit-numeric-conversions-table.md (hunk 2 above)
Comment (volkan-yazilim) [reply docs/csharp/language-reference/keywords/implicit-numeric-conversions-table.md]: Thanks

**Comment by rpetrusha** [comment]
Thanks, @volkan-yazilim. I'll merge your PR as soon as the internal build completes successfully. The changes should be live on docs.microsot.com in the next day or two.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**